substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfSectionImpl.java

```diff
@@ -274,6 +276,37 @@ protected int putAsciiStringBytes(String s, int startChar, byte[] buffer, int p)
         return pos;
     }
 
+    protected static int countUTF8Bytes(String s) {
+        return countUTF8Bytes(s, 0);
+    }
+
+    protected static int countUTF8Bytes(String s, int startChar) {
+        try {
+            byte[] bytes = s.substring(startChar).getBytes("UTF-8");
```

Comment:
use the `getBytes` variant that takes a `Charset` instead of `String` (with `StandardCharsets.UTF_8`). That also avoids the ugly handling of `UnsupportedEncodingException`

Reply:
Done.